package.json

```diff
@@ -6,6 +6,7 @@
    "types": "./dist/types/index.d.ts",
    "module": "./dist/esm/index",
    "scripts": {
+      "prepare": "grunt build",
```

Comment:
Would this actually work? Wouldn't it require dev dependencies that wouldn't be installed on a normal `npm install`?

Reply:
Haha ... should've read the blog post you linked to.

> Git dependencies with prepare scripts will have their devDependencies installed, and npm install run in their directory before being packed.

Confirmed, it works. Brilliant!

```
jrthomer@lappy ~
$ mkdir /tmp/foo && pushd /tmp/foo && npm init -y
/tmp/foo ~ ~
Wrote to /private/tmp/foo/package.json:

{
  "name": "foo",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC"
}


jrthomer@lappy /tmp/foo
$ npm i git://github.com/silvermine/lambda-express.git#npm_prepare
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN foo@1.0.0 No description
npm WARN foo@1.0.0 No repository field.

+ @silvermine/lambda-express@0.1.0
added 6 packages from 6 contributors and audited 6 packages in 1.482s
found 0 vulnerabilities

jrthomer@lappy /tmp/foo
$ ll node_modules/\@silvermine/lambda-express/dist/
total 0
drwxr-xr-x  5 jrthomer wheel 160 Mar 21 22:16 .
drwxr-xr-x 10 jrthomer wheel 320 Mar 21 22:16 ..
drwxr-xr-x 21 jrthomer wheel 672 Mar 21 22:16 commonjs
drwxr-xr-x 21 jrthomer wheel 672 Mar 21 22:16 esm
drwxr-xr-x 13 jrthomer wheel 416 Mar 21 22:16 types
```

Reply:
I have no idea how I missed that back when I was scouring over the NPM 5.x changes!